overlay_new.py

```diff
@@ -0,0 +1,197 @@
+from abc import ABC, abstractmethod
+from time import time
+
+from canvas import Canvas, Colour
+from data import Data
+from overlay import Overlay
+from textwrap import wrap
+from typing import Callable, Tuple
+
+class Drawable(ABC):
+    @abstractmethod
+    def draw_base(self, canvas: Canvas):
+        pass
+    @abstractmethod
+    def draw_data(self, canvas: Canvas, data: Data):
+        pass
+
+class DataField(Drawable):
```

Comment:
That's a good point. I guess it depends whether we want to reuse the same components in future overlays, and standardise this whole `Drawable` thing

Reply:
I would love for this to be standardised! 

If it is possible to include these into it's own folder WITHOUT changing how we run the scripts then I would say do it in this PR! I do have a feeling that you will need to run these as a module to get it working and it may break other python scripts. Not sure though.

Reply:
This made me realise that components is probably an all-round better term to use than drawable... I've done gone and done some renaming, drawables are now components